src/app/modules/core/services/logs.service.ts

```diff
@@ -1,75 +1,50 @@
 import { Injectable } from '@angular/core';
-import { Observable, of, timer } from 'rxjs';
-import { webSocket } from 'rxjs/webSocket';
-import { concatMap, delay, delayWhen, mergeAll, retryWhen, switchMap, tap } from 'rxjs/operators';
+import { stream } from '../../core/utils/ndjson';
```

Comment:
Don't know if this is code style of prysm, but should we move local imports after library imports?

Reply:
Yep we should, my bad here